server/adapter/src/main/scala/com/eigengo/lift/adapter/AdapterMicroservice.scala

```diff
@@ -24,7 +25,7 @@ object AdapterBoot {
 
 object AdapterMicroservice extends MicroserviceApp(MicroserviceProps("Adapter")) {
 
-  override def boot(implicit system: ActorSystem, cluster: Cluster): BootedNode = {
+  override def boot(config: Config)(implicit system: ActorSystem, cluster: Cluster): BootedNode = {
```

Comment:
Whilst I don't have any objections to passing in `config` as a parameter, I do note that it is also possible to extract this value via `system.settings.config`. So, is there a specific reason for having it as an extra argument?